src/magmaviz/boxplot.py

```diff
@@ -28,6 +31,10 @@ def boxplot(df, x, y, facet=False):
     """
 
     # checking that type of column name for x is a string
```

Comment:
This comment is not for the function defined one line below. Should be moved to the right place for clarity.

Reply:
agreed, changes reflected in latest commit